lambda.tf

```diff
@@ -49,22 +44,25 @@ resource "aws_iam_role" "lambda_role" {
   assume_role_policy = file("${path.module}/policies/LambdaBasicExecution.json")
 }
 
-# resource "aws_iam_role_policy" "lambda_iam_role_policy" {
-#   name   = "lambda_iam_role_policy"
-#   role   = aws_iam_role.lambda_role.name
-#   policy = file("${path.module}/policies/Watchlog.json")
-# }
-
-resource "aws_iam_role_policy" "lambda_iam_role_policy" {
-  name   = "lambda_iam_role_policy"
-  role   = aws_iam_role.lambda_role.name
-  policy = data.template_file.example.rendered
+data "aws_iam_policy_document" "policy_lambda_logs" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+    effect    = "Allow"
+    resources = ["arn:aws:logs:*:*:*"]
+  }
 }
+resource "aws_iam_role_policy" "log_policy" {
+  name   = "lambda_log_policy"
+  role   = aws_iam_role.lambda_role.id
+  policy = data.aws_iam_policy_document.policy_lambda_logs.json
+}
+resource "aws_iam_role_policy_attachment" "lambda_iam_role_policy_attachment" {
+  for_each = var.lambda_policy_arn
 
-data "template_file" "example" {
-  template = file("${path.module}/policies/Watchlog.json.tpl")
-
-  vars = {
-    resource = "${var.iam_source_name.default}"
-  }
+  role       = aws_iam_role.lambda_role.name
+  policy_arn = var.lambda_policy_arn[each.key] #element(var.lambda_policy_arn, count.index)
```

Comment:
`each.value` = `var.lambda_policy_arn[each.key]` by definition 
delete the comment

Reply:
Done, commit: 8f78c78f9c18aca02ee307e6c070f7c8905196e4